pkg/controller/ttlafterfinished/ttlafterfinished_controller.go

```diff
@@ -238,7 +238,7 @@ func (tc *Controller) processJob(key string) error {
 		PropagationPolicy: &policy,
 		Preconditions:     &metav1.Preconditions{UID: &fresh.UID},
 	}
-	klog.V(4).Infof("Cleaning up Job %s/%s", namespace, name)
+	klog.V(4).InfoS("Cleaning up Job", "jobNamespace", namespace, "job", name)
```

Comment:
Same as above

Reply:
This can actually use `klog.KRef`, thanks for catching it.